_posts/2022-12-29-release.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Release 2022.52
+author: priya
+date: 2022-12-29
+category: Jekyll
+layout: post
+---
+
+### Enhancement
+
+#### Add flexibility to show/hide joins and views in dataflow map
+Mammoth now added checkboxes to the dataflow map that allow you to show or hide joins and barren views(views with no children)
+This allows users to customize the visualization of the dataflow according to their specific needs."
+
+### Defect fixes
+
+* The export action does not auto-run on the data updates
```

Comment:
In some scenarios, the export task would not run by itself on data updates. This has been fixed now.

Reply:
done